internal/viridian/custom_class.go

```diff
@@ -0,0 +1,107 @@
+package viridian
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+)
+
+func (a API) ListCustomClasses(ctx context.Context, cluster string) ([]CustomClass, error) {
```

Comment:
Could you move these methods to `api.go` next to `func (a API) ListClusters` ?
Same comment for other `API` methods.